docs/basicusage.rst

```diff
@@ -20,13 +19,15 @@ First create a new ``config.ini`` file, and add the ``[main]`` section:
     daemon = true
     sleep = 900
 
-Set ``daemon`` to ``true`` if you want ThreatIngestor to constantly watch
-your sources in a loop; set it to ``false`` if you want to run it manually,
-or via cron or some other scheduler. Set ``sleep`` to the number of seconds
-to wait between each check - this will be ignored if you disable daemon mode.
+.. _daemon:
+Configure ``daemon`` to run continuously or manually. If you set ``daemon`` to ``true``, ThreatIngestor will watch
+your sources in a loop; set it to ``false`` to run manually, or via cron or some other scheduler. 
+Set ``sleep`` to the number of seconds to wait between each check - this will be ignored if you daemon mode is false.
 Don't set the sleep too low, or you may run into rate limits or other issues.
 If in doubt, keep this above 900 (fifteen minutes).
 
+.. _source:
+.. _operator:
```

Comment:
The format of these is correct, but these tags work throughout the entire documentation base, so they should probably be more specific to avoid name conflicts.

Reply:
These are local links, meant to jump to the appropriate section